tests/page-objects/default/sms/messages.wdio.page.js

```diff
@@ -65,22 +96,51 @@ const getMessageLoadingStatus = async () => {
   return await (await messagesLoadingStatus()).getText();
 };
 
+const sendReply = async (message) => {
+  await (await replyMessage()).setValue(message);
+  await (await replyMessageActions()).waitForExist();
+  await (await submitReplyBtn()).waitForClickable();
+  await (await submitReplyBtn()).click();
+};
+
+const replyAddRecipients = async (message) => {
+  await (await replyMessage()).setValue(message);
+  await (await replyMessageActions()).waitForExist();
+  await (await addRecipient()).waitForClickable();
+  await (await addRecipient()).click();
+  await (await $(SEND_MESSAGE_MODAL)).waitForDisplayed();
+};
+
+const getAmountOfMessages = async () => {
+  await (await $(MESSAGE_CONTENT)).waitForDisplayed();
+  const listedMessages = (await messages());
+  return listedMessages.length;
+};
+
+const getMessagesModalDetails = async () => {
+  await (await $(SEND_MESSAGE_MODAL)).waitForDisplayed();
+  return {
+    recipient: await $(`${SEND_MESSAGE_MODAL} .select2-selection__choice`).getText(),
+    message: await messageText().getAttribute('ng-reflect-model'),
+  };
+};
+
 module.exports = {
+  //messageByIndex,
+  openMessage,
+  getMessageInListDetails,
+  getMessageHeader,
+  getMessageContent,
+  sendMessage,
+  sendReplyNewRecipient,
+  sendMessageToContact,
   exportMessages,
-  messageByIndex,
-  waitForMessagesInLHS,
-  listMessageHeading,
-  listMessageSummary,
-  listMessageLineage,
-  clickLhsEntry,
   messageDetailsHeader,
-  messageContentText,
-  messageContentIndex,
-  messageDetailStatus,
   messagesList,
   messageText,
```

Comment:
I don't think we need to export this anymore since it is only used in this file.